js/components/atlas-state.js

```diff
@@ -60,6 +60,8 @@ define(['knockout', 'lscache', 'services/job/jobDetail',  'assets/ohdsi.util', '
 		analysisPath: null,
 		selectedId: ko.observable(null),
 		comparisons: ko.observableArray(),
+		notificationSourceId: ko.observable(),
```

Comment:
Do we need to store the ids in the global state? Cannot we just pass them as GET params in a link? I'm just a bit afraid of forgetting to clean up those and getting anomalies (the stateless mechanism is more bulletproof)

Reply:
Fixed